### PR TITLE
history position fix

### PR DESCRIPTION
**Pull request by sah-huawei**
This PR has two things in it:

1) A fix for a bug in `genhistories.py` with how it shifts vehicle positions from the front of the car to the center (Issue #1316).  This depends on heading, but the original code just assumed the car was oriented in the x-direction.  (This worked originally b/c the original i-80 map was oriented along the x-axis.)

2) Increasing the minimum version of `numpy` that's required in `setup.py` because one of the APIs we use in `genhistories.py` (`sliding_window_view()`) wasn't introduced until version `1.20`.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
         "click>=8.0.3",  # used in scl
         "eclipse-sumo==1.10.0",  # sumo
         "gym==0.19.0",
-        "numpy>=1.19.5",
+        "numpy>=1.20.0",
```
Comment (Gamenot) [inline setup.py:30]: Why is this needed?

**Review by Gamenot — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (sah-huawei) [reply setup.py:30]: It's the second thing I wrote about that in the PR description:  we've been using `sliding_window_view()` in `genhistories.py` for a while and `numpy` has been in `setup.py` for a while without a minimum version.  But recently, we added a minimum version of `numpy` there which is too low such that, if you reinstall and get `1.19.5` (our stated minimum version) then it won't support `sliding_window_view()`.  I encountered this when trying to test the fix above for Patrick.

**Review by qianyi-sun — APPROVED** [review]

**Review by RutvikGupta — APPROVED** [review]
With `numpy` version set to 1.19.5 again, it looks ready to merge.

File: setup.py (hunk 1 above)
Comment (sah-huawei) [reply setup.py]: ok, due to issue #1321, I've reverted the `numpy` version to `1.19.5`.
